databases/week3/Week3Homework.sql

```diff
@@ -0,0 +1,205 @@
+SET NAMES utf8mb4;
+
+CREATE DATABASE meals_sharing;
+use meals_sharing;
```

Comment:
Kill this line 4. We only use this to select a specific DB in the MySQL workbench.

Reply:
Done